rotkehlchen/globaldb/cache.py

```diff
@@ -1,75 +1,92 @@
 """Functions dealing with the general_cache table of the Global DB"""
+import sqlite3
 from collections.abc import Iterable
 from typing import Optional, Union
+
 from rotkehlchen.chain.evm.constants import ZERO_ADDRESS
 from rotkehlchen.chain.evm.types import string_to_evm_address
-
 from rotkehlchen.db.drivers.gevent import DBCursor
+from rotkehlchen.errors.misc import InputError
 from rotkehlchen.types import ChecksumEvmAddress, GeneralCacheType, Timestamp
 from rotkehlchen.utils.misc import ts_now
 
+UNIQUE_CACHE = 'unique_cache'
+GENERAL_CACHE = 'general_cache'
```

Comment:
Those need to match the table name right? Then it should be something that reminds us that like `UNIQUE_CACHE_TABLE_NAME`